config/jobs/kubernetes/sig-testing/conformance-e2e.yaml

```diff
@@ -47,6 +47,63 @@ presubmits:
       testgrid-dashboards: sig-testing-misc
       description: Runs conformance tests using kind and the conformance image
   - name: pull-kubernetes-conformance-kind-ipv6
+    branches:
+    - master
+    always_run: false
+    skip_report: false
+    max_concurrency: 8
```

Comment:
we may need to reduce these with more jobs and not more infra ...

Reply:
copy paste :)